.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -6,12 +6,12 @@ body:
   - type: markdown
     attributes:
       value: |
-         Thanks for thinking of a way to improve Matplotlib. Please search the issues for relevant feature requests before creating a new feature request.  
+         Please search the [issues](https://github.com/matplotlib/matplotlib/issues) for relevant feature requests before creating a new feature request.
   - type: textarea
     id: problem
     attributes:
       label: Problem
-      description: Please provide a clear and concise description of the problem this feature will solve. 
+      description: A concise description of the problem this feature will solve.
```

Comment:
```suggestion
      description: Briefly describe the problem this feature will solve. (2-4 sentences)
```

Reply:
I think I agree with @story645 that the imperative voice is a little nicer here than the passive.